modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer.js

```diff
@@ -64,6 +69,18 @@ export default class MultiIconLayer extends IconLayer {
     }
   }
 
+  draw({uniforms}) {
+    const {cutoff, fontSmoothing, sizeScale} = this.props;
+
+    super.draw({
+      uniforms: Object.assign({}, uniforms, {
+        cutoff,
+        // TODO: handle for individual sizes using this.props.getSize()
```

Comment:
This needs to be handled in the vertex shader and calculated for each object.
Setting `getSize: 1, sizeScale: 10` and `getSize: 10, sizeScale: 1` should not result in any difference.